File: LeetCode Practice/Python_Practive/How Many Numbers Are Smaller Than the Current Number.py
class Solution:
    def smallerNumbersThanCurrent(self, nums: List[int]) -> List[int]:
        ans = []
        count = 0

        for i, inum in enumerate(nums):
            count = 0
            for j, jnum in enumerate(nums):
                if jnum != inum and jnum < inum:
                    count += 1
            
            ans.append(count)
        
        return ans

File: LeetCode Practice/Javascript_Practice/How Many Numbers Are Smaller Than the Current Number.js
/**
 * @param {number[]} nums
 * @return {number[]}
 */
var smallerNumbersThanCurrent = function (nums) {
    let n = nums.length;
    let ans = [];

    for (let i = 0; i < n; i++) {
        let num = nums[i];
        let count = 0;

        for (let j = 0; j < n; j++) {
            if (nums[j] != num && nums[j] < num) {
                count = count + 1;
            }
        }

        ans.push(count);
    }

    return ans;
};
